components/cqrs/cqrs_test.go

```diff
@@ -22,23 +24,23 @@ func TestCQRS(t *testing.T) {
 	router, cqrsFacade := createRouterAndFacade(ts, t, captureCommandHandler, captureEventHandler)
 
 	pointerCmd := &TestCommand{ID: watermill.NewULID()}
-	require.NoError(t, cqrsFacade.CommandBus().Send(pointerCmd))
+	require.NoError(t, cqrsFacade.CommandBus().Send(context.Background(), pointerCmd))
 	assert.EqualValues(t, []interface{}{pointerCmd}, captureCommandHandler.HandledCommands())
 	captureCommandHandler.Reset()
 
 	nonPointerCmd := TestCommand{ID: watermill.NewULID()}
-	require.NoError(t, cqrsFacade.CommandBus().Send(nonPointerCmd))
+	require.NoError(t, cqrsFacade.CommandBus().Send(context.Background(), nonPointerCmd))
 	// command is always unmarshaled to pointer value
 	assert.EqualValues(t, []interface{}{&nonPointerCmd}, captureCommandHandler.HandledCommands())
 	captureCommandHandler.Reset()
 
 	pointerEvent := &TestEvent{ID: watermill.NewULID()}
-	require.NoError(t, cqrsFacade.EventBus().Publish(pointerEvent))
+	require.NoError(t, cqrsFacade.EventBus().Publish(context.Background(), pointerEvent))
 	assert.EqualValues(t, []interface{}{pointerEvent}, captureEventHandler.HandledEvents())
 	captureEventHandler.Reset()
 
 	nonPointerEvent := TestEvent{ID: watermill.NewULID()}
-	require.NoError(t, cqrsFacade.EventBus().Publish(nonPointerEvent))
+	require.NoError(t, cqrsFacade.EventBus().Publish(context.Background(), nonPointerEvent))
 	// event is always unmarshaled to pointer value
 	assert.EqualValues(t, []interface{}{&nonPointerEvent}, captureEventHandler.HandledEvents())
```

Comment:
we could probably have separate unit tests that check if the context is propagated in `CommandBus.Send` and `EventBus.Publish`.

It's a formality, but it's a part of the contract now, I guess?